CONTRIBUTING.md

```diff
@@ -52,6 +75,15 @@ for development
     cd cvat-core && npm install && \
     cd ../cvat-ui && npm install && npm start
     ```
+    > Note for Mac users
+    >
+    > If you faced with error
```

Comment:
Probably better to collect links like [Node Sass does not yet support your current environment...](https://stackoverflow.com/questions/37415134/error-node-sass-does-not-yet-support-your-current-environment-windows-64-bit-w)
The tutorial will be very hard to read if it has a lot of notes like this one.